### DISCUSSION TITLE
Not getting estimates for non-brain tissue

### DISCUSSION THREAD

**Discussion by Giri5A136** [simnibs/simnibs#308] [8. Simulation workflows]
Hello, when I run the modeling, I'm only getting the estimated E and J for grey matter. I would like to see the estimates for white matter, skull, eye balls etc. Appreciate any help with what change in settings are needed. Thanks.

**Comment by TorgeW** [comment]
If you have issue displaying these results in gmsh refer to:
https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html

If you miss the fields in the written out nifties you can use the following options in the scripts,
Adding this to the scripts will add white matter (tissue type 1), gray matter (tissue type 2) and csf (tissue type 3) to the nifti output
```
S.map_to_vol = True     #  Save as nifti volume
S.map_to_MNI = True     #  Save in MNI space
S.tissues_in_niftis = [1,2,3]

or matlab:
S.map_to_vol = true;    %  Save as nifti volume
S.map_to_MNI = true;    %  Save in MNI space
S.tissues_in_niftis = [1,2,3];
```
